CHANGELOG.md

```diff
@@ -1,10 +1,75 @@
-## vUNRELEASED
+## v1.4.0
 
 #### New features and UX changes
 
-- Add `--user`/`--group` option to rkt run/prepare ([#2419](https://github.com/coreos/rkt/pull/2419)). This option allows overriding the user/group specified in the image manifest.
+- config: add config subcommand ([#2405](https://github.com/coreos/rkt/pull/2405)). This new subcommand prints the current rkt configuration. It can be used to get i.e. authentication credentials.
 
-- Ensure that the initial name and labels used for discovery match the name and labels in the Image Manifest as specified in the appc spec ([#2311](https://github.com/coreos/rkt/pull/2311)). Users wanting the latest image should use `rkt prepare/run/fetch example.com/aci` without any labels. If the discovery server supports the "latest" pattern, the user can bypass a locally cached image in the store and fetch an updated image using `rkt prepare/run/fetch --no-store example.com/aci` option.
+- run: add user/group app flags ([#2419](https://github.com/coreos/rkt/pull/2419)). This adds the `--user`/`--group` options to rkt run/prepare allowing to override the user/group specified in the image manifest.
+
+- gc: Add flag 'mark-only' to mark garbage pods without deleting them ([#2400](https://github.com/coreos/rkt/pull/2400), [2402](https://github.com/coreos/rkt/pull/2402)). This new flag moves exited/aborted pods to the exited-garbage/garbage directory but does not delete them. A third party application can use `rkt gc --mark-only=true` to mark exited pods as garbage without deleting them.
+
+- kvm: Add support for capabilities limitation ([#2222](https://github.com/coreos/rkt/pull/2222)). By default kvm flavor has got enabled every capability inside pod. This patch adds support for a restricted set of capabilities inside a rkt-kvm pod.
+
+- kvm: new volumes support ([#2328](https://github.com/coreos/rkt/pull/2328)). This provides new volumes support based on bind mounting volumes in time of rootfs preparation for the kvm flavor.
```

Comment:
we should say which bug it was fixing. Volumes were already implemented in kvm, but it was not working well in some cases. Was it when the directory name was too long or something?